PIL/Image.py

```diff
@@ -2206,6 +2206,8 @@ def fromqpixmap(im):
     # first two members of shape are set to one
     # ((1, 1), "|b1"): ("1", "1"), # broken
     ((1, 1), "|u1"): ("L", "L"),
+    ((1, 1), "<u2"): ("L", "I;16"),
+    ((1, 1), ">u2"): ("L", "I;16B"),
     ((1, 1), "|i1"): ("I", "I;8"),
```

Comment:
The actual datatype is a uint16, so there's no dataloss with the uint8 issue you talk about.